DEVELOPMENT.md

```diff
@@ -29,14 +35,17 @@ While not strictly necessary (many/most of our contributors do not do this!),
 if you want to exactly reproduce our CI builds you'll want to do the same:
 
 ```
+$ rm Gemfile.lock
 $ bundle install --standalone --path <current_lib>/bundle
```

Comment:
Maybe instead recommend `bundle install --force ..` ?

Reply:
Thats not what we do in CI